cli/cmd/install.go

```diff
@@ -182,6 +186,17 @@ func validateAndBuildConfig(options *installOptions) (*installConfig, error) {
 		profileSuffixes = "svc.cluster.local."
 	}
 
+	jsonMarshaler := jsonpb.Marshaler{}
```

Comment:
In other places where we use the jsonpb marshaler, we use:

```
jsonpb.Marshaler{EmitDefaults: true, Indent: "  "}
```

I think we should use `EmitDefaults` here too, but I'm on the fence about `Intent`.